components/clp-mcp-server/tests/test_validator.py

```
@@ -0,0 +1,15 @@
+
+from clp_mcp_server.validator import validate_query
+
+
+def test_validate_query():
```

Ruff (0.14.3)

Missing docstring in public function

(D103)